packaging/datadog-agent/source/install_agent.sh

```diff
@@ -158,6 +160,24 @@ If the cause is unclear, please contact Datadog support.
 "
     $sudo_cmd apt-get install -y --force-yes datadog-agent
     ERROR_MESSAGE=""
+elif [ $OS = "SUSE" ]; then
+  UNAME_M=$(uname -m)
+  if [ "$UNAME_M"  == "i686" -o "$UNAME_M"  == "i386" -o "$UNAME_M"  == "x86" ]; then
+    printf "The Datadog Agent installer is only available for 64 bit SUSE Enterprise machines."
+    exit;
+  else
+      ARCHI="x86_64"
+  fi
+
+  echo -e "\033[34m\n* Installing YUM Repository for Datadog\n\033[0m"
+  $sudo_cmd sh -c "echo -e '[datadog]\nname=datadog\nenabled=1\nautorefresh=0\nbaseurl=https://yum.datad0g.com/suse-testing/x86_64\ntype=rpm-md\ngpgcheck=1\nrepo_gpgcheck=0\ngpgkey=https://yum.datadoghq.com/DATADOG_RPM_KEY.public' > /etc/zypp/repos.d/datadog.repo"
```

Comment:
Also, and more importantly, we need to update the `baseurl` to the prod/stable repo before we merge right?